tests/health_checks/NDv4/dcgmi-check.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+#Turn on persistence mode.
+sudo nvidia-smi -pm 1 > /dev/null
```

Comment:
I did try to find a way to do this before but it didn't seem like there was an easy way. I will look into it again though as it would be ideal, yeah.